invenio_s3/multipart_client.py

```diff
@@ -157,6 +157,14 @@ def complete_multipart_upload(self, parts):
             },
         )
 
+        if not response.get("ETag", None):
+            # Fix for Ceph < v18.2.4, where ETag is empty in complete multipart responses
+            # The checksum is still computed and can be fetched by HEAD object request
+            etag = self.s3_client.head_object(Bucket=self.bucket, Key=self.key).get("ETag")
+            response["ETag"] = etag
+
+        return response
```

Comment:
I remember I had this problem myself some time ago! So this makes perfect sense to me!